### PR TITLE
doc: Update Requirements section in samples

### PR DESCRIPTION
**Pull request by umapraseeda**
Update the Requirements section of few samples
by adding the table that lists the supported
development kits.

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: samples/debug/ppi_trace/README.rst [hunk 1]
```diff
@@ -28,12 +28,13 @@ When the low-frequency crystal is ready, an ``NRF_CLOCK_EVENT_LFCLKSTARTED`` eve
...
-* One of the following development boards:
+The sample supports the following development kits:
 
-  * |nRF9160DK|
-  * |nRF52840DK|
-  * |nRF52DK|
-* Logic analyzer
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52dk_nrf52832, nrf52840dk_nrf52840, nrf9160dk_nrf9160
```
Comment (ru-fu) [inline samples/debug/ppi_trace/README.rst:35]: nrf9160dk_nrf9160 or nrf9160dk_nrf9160**ns**?

File: samples/nrf9160/lwm2m_carrier/README.rst [hunk 2]
```diff
@@ -12,11 +12,13 @@ The LwM2M carrier sample demonstrates how to run the :ref:`liblwm2m_carrier_read
 Requirements
 ************
 
-* The following development board:
+The sample supports the following development kit:
 
-  * |nRF9160DK|
+  .. table-from-rows:: /includes/sample_board_rows.txt
```
Comment (ru-fu) [inline samples/nrf9160/lwm2m_carrier/README.rst:17]: Should this be indented?

Comment (umapraseeda) [reply samples/nrf9160/lwm2m_carrier/README.rst:17]: removed the space.

File: samples/debug/ppi_trace/README.rst (hunk 1 above)
Comment (umapraseeda) [reply samples/debug/ppi_trace/README.rst:35]: nrf9160dk_nrf9160

**Review by ru-fu — APPROVED** [review]
Looks good to me!

**Review by lemrey — APPROVED** [review]
Thanks

**Review by nordic-krch — APPROVED** [review]

File: samples/event_manager/README.rst [hunk 3]
```diff
@@ -34,11 +34,11 @@ Statistics (``stats.c``):
 Requirements
 ************
 
-* One of the following development boards:
+The sample supports the following development kit:
```
Comment (anangl) [inline samples/event_manager/README.rst:37]: kit**s**

**Review by anangl — APPROVED** [review]

**Review by doki-nordic — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**